examples/03_connectivity/plot_atlas_comparison.py

```diff
@@ -90,17 +87,79 @@
 # using connectome measure object
 mean_correlation_matrix = connectome_measure.mean_
 
+# useful for plotting connectivity interactions on glass brain
+from nilearn import plotting
+
 # grab center coordinates for atlas labels
 coordinates = plotting.find_parcellation_cut_coords(labels_img=yeo["thick_17"])
 
 # plot connectome with 80% edge strength in the connectivity
-plotting.plot_connectome(
-    mean_correlation_matrix,
-    coordinates,
-    edge_threshold="80%",
-    title="Yeo Atlas 17 thick (func)",
+left_connectome = plotting.plot_connectome(
+    mean_correlation_matrix, coordinates, edge_threshold="80%"
 )
 
+# %%
+# Note that the approach above will extract time series and compute a
+# single connectivity matrix for both hemispheres. However, the connectome
+# is plotted only for the left hemisphere. If your aim is to compute and plot
+# hemisphere-wise connectivity, you can follow the example below.
+# First, create a separate atlas image for each hemisphere:
+
+import nibabel as nb
+import numpy as np
+
+from nilearn.image import get_data
+from nilearn.image.resampling import coord_transform
+
+# load the atlas image first
+label_image = nb.load(yeo["thick_17"])
+
+# extract the affine matrix of the image
+labels_affine = label_image.affine
+
+# generate image coordinates using affine
+x, y, z = coord_transform(0, 0, 0, np.linalg.inv(labels_affine))
+
+# generate an separate image for the left hemisphere
+# left/right split is done along x-axis
+left_hemi = get_data(label_image).copy()
+left_hemi[: int(x)] = 0
+label_image_left = nb.Nifti1Image(left_hemi, labels_affine)
```

Comment:
I'd rather write it:
```
from nilearn.image import new_img_like
label_image_left = new_img_like(label_image, left_hemi)
```

Reply:
`new_img_like` can also take the affine:

```python
from nilearn.image import new_img_like 
label_image_left = new_img_like(label_image, left_hemi, labels_affine)
```